addons/mail/models/mail_activity.py

```diff
@@ -127,9 +127,27 @@ def default_get(self, fields):
         'Document Name', compute='_compute_res_name', compute_sudo=True, store=True,
         help="Display name of the related document.", readonly=True)
     # activity
+    @api.model
+    def default_get_activity_type(self):
+        activity_type = self.env["mail.activity.type"]
```

Comment:
Just for convention, variables being an Odoo class are labelled like ActivityType, to distinguish from activity_type which would be a record set.